src/postgres/event-handlers/CommentReactionCreated.ts

```diff
@@ -1,24 +1,30 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
-import { substrate } from '../../substrate/subscribe';
+import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial';
 import { insertActivityForCommentReaction } from '../insert-activity';
 import { insertNotificationForOwner } from '../notifications';
-import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+import { SubstrateEvent } from '../../substrate/types';
 
-export const onCommentReactionCreated: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+export const onCommentReactionCreated = async (eventAction: SubstrateEvent, post: Post) => {
   const { data } = eventAction;
   const follower = data[0].toString();
-  const commentId = data[1] as CommentId;
-  const comment = await substrate.findComment(commentId);
-  if (!comment) return;
+  const commentId = data[1] as PostId;
 
-  const ids = [ comment.post_id, commentId ];
-  const account = comment.created.account.toString();
-  const count = (comment.upvotes_count.toNumber() + comment.downvotes_count.toNumber()) - 1;
-  const activityId = await insertActivityForCommentReaction(eventAction, count, ids, account);
+  const { 
+    extension: { asComment: { parent_id, root_post_id } },
+    created: { account },
```

Comment:
You don't need to get `account`. You can get comment `author` from blokchain event